guides/source/active_storage_overview.md

```diff
@@ -391,6 +391,19 @@ class Message < ApplicationRecord
   has_many_attached :images, service: :s3
 end
 ```
+### Attaching A Single Blob To Multiple Records
+
+Sometimes you want to attach the same blob to multiple records. One situation where
+this might occur is if you want to duplicate a record. Active Storage supports multiple
+records attached to a single blob. To do so, attach the original blob to a new record:
+
+```ruby
+@new_record.image.attach(@original.image.blob)
```

Comment:
```suggestion
@new_message.images.attach(@original_message.images.blobs)
```

Reply:
@vipulnsward I don't have a strong opinion here but do you think there'd be value in splitting this up into has_one_attached versus attaching to an has_many_attached? They work similarly I assume (never personally used has_many_attached so not sure) but it might not be obvious that both work.

Reply:
Sure, let's expand the example and show both. I posted this correction to match the other examples. The example of `@new_record.image` here does not have a context.